.pubnub.yml

```diff
@@ -1,5 +1,12 @@
 --- 
 changelog:
+  - 
+    changes: 
+      - 
+        text: "Objects Sorting"
+        type: feature
+    date: April 10, 20  
+    version: v4.8.2 
```

Comment:
Missing the root property version increase `4.8.2`

Reply:
Got this!